code/__DEFINES/emotes_defines.dm

```diff
@@ -75,3 +77,12 @@
 
 /// List of emotes useable by ghosties
 #define USABLE_DEAD_EMOTES list("*flip", "*spin")
+
+// Strings used for the rock paper scissors emote and status effect
+#define RPS_EMOTE_ROCK 		"rock"
+#define RPS_EMOTE_PAPER 	"paper"
+#define RPS_EMOTE_SCISSORS 	"scissors"
+
+#define RPS_EMOTE_THEY_WIN		"aww"
+#define RPS_EMOTE_WE_WIN		"yay"
+#define RPS_EMOTE_TIE			"tie"
```

Comment:
github will vaporize the indents if i try to edit them, but can you try to pull the quotes over a tad, they look out of place.
Also what is with those names, i hope the quoted parts aren't use anywhere? im not awfully sure what the second part of defines are for.

Reply:
Nah, the second part of the defines is pretty much arbitrary, as long as they're unique here. I just figured I'd get silly with it.

I can probably move the tabs over tho